manifests/server.pp

```diff
@@ -54,6 +54,19 @@
   $manage_pg_ident_conf       = $postgresql::params::manage_pg_ident_conf,
   $manage_recovery_conf       = $postgresql::params::manage_recovery_conf,
 
+  $config_entries  = {},
+  $databases       = {},
+  $database_grants = {},
+  $dbs             = {},
+  $extensions      = {},
+  $pg_hba_rules    = {},
+  $pg_ident_rules  = {},
+  $recovery        = {},
+  $roles           = {},
+  $schemas         = {},
+  $table_grants    = {},
+  $tablespaces     = {},
+
```

Comment:
The related discussion to come up with this pattern is in https://github.com/voxpupuli/community-triage/blob/master/modules/notes/2017-03-02.md . tl;dr is that "this isn't a great pattern to be stuck on, but it seems like most people still have to move through it before they arrive at profile abstraction patterns so how can we facilitate that path of learning?"